app.js

```diff
@@ -32,6 +33,13 @@ config.session.store = new PostgresStore({ db: sequelize });
 app.engine('html', require('ejs').renderFile);
 
 app.use(session(config.session));
+app.use(passport.initialize());
+app.use(passport.session());
+app.use((req, res, next) => {
+  res.locals.user = req.user;
+  next();
+});
```

Comment:
moved `passport` here to the main app instead of adding it to each individual router. Added a small middleware to put `user` in `res.locals` -- this used to be done in `api/policies/passport.js`, which has been deleted.